runtime/onert/backend/cpu/CMakeLists.txt

```diff
@@ -1,5 +1,3 @@
-set(LIB_ONERT_BACKEND_CPU onert_backend_cpu)
-
```

Comment:
Do we need this code?

Reply:
If you mean the code in `backend/CMakeLists.txt`, maybe yes.

Reply:
Sorry, I misunderstood. :(